test/rankcorr.jl

```diff
@@ -26,17 +26,63 @@ c22 = corspearman(x2, x2)
 
 # corkendall
 
-@test corkendall(x1, y) ≈ -0.105409255338946
-@test corkendall(x2, y) ≈ -0.117851130197758
+@test_throws ErrorException("Vectors must have same length") corkendall([1,2,3,4], [1,2,3])
+@test isnan(corkendall([1,2], [3,NaN]))
+@test isnan(corkendall([1,1,1], [1,2,3]))
```

Comment:
Can you also test `Inf`, `-Inf` and `-0.0`?

Reply:
That's done. Of course `Inf` and `-Inf` in input data are quite benign since they can be safely compared to finite values.